frameworks/hdfs/tests/test_sanity.py

```diff
@@ -296,7 +297,7 @@ def test_modify_app_config():
     config.check_healthy(service_name=foldered_name)
     sdk_tasks.check_tasks_updated(foldered_name, 'journal', journal_ids)
     sdk_tasks.check_tasks_updated(foldered_name, 'name', name_ids)
-    sdk_tasks.check_tasks_updated(foldered_name, 'data', journal_ids)
+    sdk_tasks.check_tasks_updated(foldered_name, 'data', data_ids)
```

Comment:
I don't believe it is the job here in PR review to perform a git spelunking expedition. I looked back:
1. one commit and this line changed, but already had incorrectly sent in `journal_ids`.
1. two commits and this line wasn't in the delta.
1. three commits and this line changed, but already had incorrectly sent in `journal_ids`.
1. four commits and this line wasn't in the delta.
1. five...not in delta.
1. six...not in delta.
...

Point being, it's been long enough wrong AND the line touched more than once, so should've been corrected long ago. It's not encouraging to put the spelunking expedition on the person who fixed the problem.

Reply:
I wasn't trying to imply that answering this was your responsibility.  I was curious how long we'd left it broken and thought this might already be known. It was an attempt to learn.